### PR TITLE
Update assetlist.json

### PR DESCRIPTION
**Pull request by Fidel2001**
TADF update token info (#4357)

### CODE REVIEW

File: terra/assetlist.json [hunk 1]
```diff
@@ -5759,6 +5759,27 @@
...
+        {
+          "denom": "tadf",
+          "exponent": 6
+        }
+      ],
+      "base": "cw20:terra19gzvkw5hft3ulp8n6ws3p9yradhc802axqal59lzxau6rqxktj9q0dwf9c",
+      "name": "Tadpole Frogztrik Token",
+      "symbol": "TADF",
+      "logo_URIs": {
+        "png": "https://https://github.com/Fidel2001/chain-registry/blob/master/terra/images/tadf.png"
```
Comment (JeremyParish69) [inline terra/assetlist.json:5781]: Please upload the image and use the URL that looks that the Chain Registry.

File: terra/assetlist.json [hunk 2]
```diff
@@ -5759,6 +5759,27 @@
...
+      }
+    },
```
Comment (JeremyParish69) [inline terra/assetlist.json:5783]: extra comma breaks the schema

**Comment by JeremyParish69** [comment]
Please use a meaningful PR title

File: terra/assetlist.json [hunk 3]
```diff
@@ -5759,6 +5759,27 @@
...
+        {
+          "denom": "tadf",
+          "exponent": 6
+        }
+      ],
+      "base": "cw20:terra19gzvkw5hft3ulp8n6ws3p9yradhc802axqal59lzxau6rqxktj9q0dwf9c",
+      "name": "Tadpole Frogztrik Token",
+      "symbol": "TADF",
+      "logo_URIs": {
+        "png": "terra/images/TADF logo"
```
Comment (JeremyParish69) [inline terra/assetlist.json:5781]: must be the full path. e.g.,
```
      "images": [
        {
          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/terra/images/tadf.png"
        }
      ],
      "logo_URIs": {
        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/terra/images/tadf.png"
      },
```

**Comment by Fidel2001** [comment]
Thanks mate. I am new here. I will try again soon

**Comment by JeremyParish69** [comment]
Why did you mark this as ready for review? The two issues are still not resolved.
1) The image is not here. You need to upload the image.
2) Please update the PR title.

**Pull request closed without merging**